client/aws/aws.go

```diff
@@ -56,7 +56,7 @@ func (e EngineType) String() string {
 type instance struct {
 	ID             string
 	ServiceCode    string `json:"servicecode"`
```

Comment:
nit: serviceCode

Reply:
AWS names this field this way.😮‍💨 I have to adopt their fault.

Reply:
sad... Add a comment to explain this?